es6-classes/example-class.js

```diff
@@ -0,0 +1,22 @@
+class ExampleClass {
+  constructor(name) {
+    this.name = name;
+  }
+
+  getClass() {
```

Comment:
Why do you have this here? It seems misplaced since `getClass` is a function that takes an object and returns the name of the class. It does this without any need for a `getClass` method.